src/main/java/org/example/CalendarManagement/thriftclients/implementation/Client.java

```diff
@@ -0,0 +1,58 @@
+package org.example.CalendarManagement.thriftclients.implementation;
+
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.transport.TSocket;
+import org.apache.thrift.transport.TTransport;
+import org.example.CalendarManagement.thriftclients.interfaces.ClientInterface;
+import org.example.CalendarThriftConfiguration.MeetingSvc;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("!test")
+public class Client implements ClientInterface
+{
+    @Override
+    public boolean cancelMeetingForRemovedEmployee(String employeeId) throws TException {
+
+        TTransport transport;
+
+        transport = new TSocket("localhost" , 9090);
+
+        transport.open();
+
+        TProtocol protocol = new TBinaryProtocol(transport);
+
+        MeetingSvc.Client client = new MeetingSvc.Client(protocol);
+
+        boolean thriftResponse = client.cancelMeetingOfRemovedEmployee(employeeId);
+
+        transport.close();
+
+        return  thriftResponse;
+    }
+
+    @Override
+    public boolean updateStatusForRemovedEmployee(String employeeId) throws TException {
+
+        TTransport transport;
+
+        transport = new TSocket("localhost" , 9090);
+
+        transport.open();
+
+        TProtocol protocol = new TBinaryProtocol(transport);
+
+        MeetingSvc.Client client = new MeetingSvc.Client(protocol);
+
+        boolean thriftResponse = client.updateStatusOfRemovedEmployee(employeeId);
+
+        transport.close();
```

Comment:
if we are opening any resource, we have to make sure it is closed. In case of any exception from the client, the transport is not being closed here. If we have many resources in this state, it would cause different kinds of leaks, eg: memory leak, connection leak, etc.,